src/applications/vaos/appointment-list/components/AppointmentsPage/ScheduleNewAppointmentRadioButtons.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react';
+import RadioButtons from '@department-of-veterans-affairs/component-library/RadioButtons';
+import { Link } from 'react-router-dom';
+import recordEvent from 'platform/monitoring/record-event';
+import { GA_PREFIX } from 'applications/vaos/utils/constants';
+
+/**
+ * React component used to conditionally render radio call-to-action buttons and start applicable workflow.
+ * @property {boolean} [showCheetahScheduleButton=false] - A boolean value to determine Whether or not to show COVID-19 vaccine option.
+ * @property {function} startNewAppointmentFlow - A function that’s called when the user starts the new appointment flow.
+ * @component
+ * @example
+ * <ScheduleNewAppointmentRadioButtons
+ *  showCheetahScheduleButton={valueFromProp}
+ *  startNewAppointmentFlow={givenFlowFromProp()}
+ * />
+ * @module appointment-list/components/ScheduleNewAppointmentRadioButtons
```

Comment:
I think we should remove `ScheduleNewAppointmentRadioButtons` from the end of this, to avoid cluttering the sidebar of the JSDoc site with every component we might add.

Reply:
Sounds good